src/main/kotlin/org/koitharu/kotatsu/parsers/site/fr/ScansMangasMe.kt

```diff
@@ -0,0 +1,193 @@
+package org.koitharu.kotatsu.parsers.site.fr
+
+import kotlinx.coroutines.coroutineScope
+import okhttp3.Headers
+import org.json.JSONArray
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import org.koitharu.kotatsu.parsers.MangaLoaderContext
+import org.koitharu.kotatsu.parsers.MangaSourceParser
+import org.koitharu.kotatsu.parsers.PagedMangaParser
+import org.koitharu.kotatsu.parsers.config.ConfigKey
+import org.koitharu.kotatsu.parsers.model.*
+import org.koitharu.kotatsu.parsers.network.UserAgents
+import org.koitharu.kotatsu.parsers.util.*
+import java.util.*
+
+@MangaSourceParser("SCANS_MANGAS_ME", "Scans Mangas Me", "fr")
+internal class ScansMangasMe(context: MangaLoaderContext) :
+	PagedMangaParser(context, MangaSource.SCANS_MANGAS_ME, 1000000) {
```

Comment:
`1000000`?
Does it means this source has only one page?
I guess in these cases might be better to use `MangaParser` and return an empty list if `offset !=0`

Reply:
Yes, it's only one page, but I put that in so as not to be emersed during the parse.
I let you correct it I could not do it before several days.
Tanks

Reply:
Finally I was able to do it quickly evnet to take care of something else, I'm not sure it's what you ask but do not hesitate to change if necessary.